lib/iris/fileformats/netcdf.py

```diff
@@ -490,21 +490,26 @@ def _set_attributes(attributes, key, value):
         attributes[str(key)] = value
 
 
-def _load_cube(engine, cf, cf_var, filename):
-    """Create the cube associated with the CF-netCDF data variable."""
+def get_actual_dtype(cf_var):
```

Comment:
@djkirkham Let's make this private. It's more of a utility function, even though it's used outside in the connective tissue of the PyKE rules.